### DISCUSSION TITLE
Wii Ext BlueRetro work with Loopy 3DS Controller Board?

### DISCUSSION THREAD

**Discussion by fvig2001** [darthcloud/BlueRetro#1134] [Q&A]
Hi

So I have already modded my New 3DS XL to have the controller board and I can control it using my Wii Classic Controller via female port->6pin connector.

I have a few questions:

1. Assuming the power is provided externally via USB-C (explicitly mentioned in controller board documents), will this work with my modded 3DS?
2. On the diagram, is it correct that it's a male Wii connector?

Thanks

**Comment by darthcloud** [comment]
You will have to try. It's known to work with Wiimote and SNESS mini so far.
